documentation/docs/libraries/java/troubleshooting.md

```diff
@@ -0,0 +1,11 @@
+# Troubleshooting
+
+
+## StackExchange
+
+> [https://iota.stackexchange.com](https://iota.stackexchange.com/)
```

Comment:
@kwek20 maybe this `>` is the problem, but also not sure

Reply:
@lucas-tortora do you maybe know what exactly is the issue, could it be this `>`?

Reply:
@kwek20 , since we're using docusarus now, you can use [admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions) if you want.